wandb/env.py

```diff
@@ -321,8 +325,16 @@ def get_magic(default: Optional[str] = None, env: Env = None) -> Optional[str]:
     return val
 
 
+def get_data_dir(env: Env = None) -> str:
+    default_dir = appdirs.user_data_dir("wandb")
+    if env is None:
+        env = os.environ
+    val = env.get(DATA_DIR, default_dir)
+    return val
+
+
 def get_cache_dir(env: Env = None) -> str:
-    default_dir = os.path.expanduser(os.path.join("~", ".cache", "wandb"))
+    default_dir = appdirs.user_cache_dir("wandb")
```

Comment:
Note: this resolves to the same path on Linux, but changes the cache location on Windows and Darwin.